dbbackup/management/commands/_base.py

```diff
@@ -4,7 +4,7 @@
 
 class BaseDbBackupCommand(LabelCommand):
     option_list = BaseCommand.option_list + (
-        make_option("--noinput", action='store_false', dest='interactive', default=True,
+        make_option("--noinput", action='store_true', default=False,
```

Comment:
Could you revert this modif ? It was like django option: https://github.com/django/django/blob/53d28f83390aed98fab4eed187eaf1edaa92c6d0/django/contrib/auth/management/commands/createsuperuser.py#L36